README.md

```diff
@@ -1916,6 +1916,50 @@ error! 'Unauthorized', 401, 'X-Error-Detail' => 'Invalid token.'
 
 ## Routes
 
+To define routes you can use the `route` method or the shorthands for the HTTP verbs. To define a route that accepts any route set to `:any`.
+Parts of the path that are denoted with a colon will be interpreted as route parameters.
+
+```ruby
+route :get, 'status' do
+end
+
+# is the same as
+
+get 'status' do
+end
+
+# is the same as (!)
```

Comment:
What's `!` here?

Reply:
I wanted to somehow raise attention that this will not result in a route parameter. But that was probably too subtle so I now extended the example.
Is this better now?